AUTHORS

```diff
@@ -39,4 +39,3 @@ Dean Elbaz <elbaz.dean@gmail.com>
 Mike Berman <evencode@gmail.com>
 Dmitriy Fedorenko <c0va23@gmail.com>
 Zach Marcantel <zmarcantel@gmail.com>
-James Maloney <jamessagan@gmail.com>
```

Comment:
Without this rollback tests fall. See: https://travis-ci.org/gocql/gocql/builds/47227393